sourcemash/frontend/static/js/views/items_view.js

```diff
@@ -28,6 +28,7 @@ Sourcemash.Views.ItemsView = Backbone.View.extend({
 
         // Render parent view
         this.$el.html(this.template({ model: this.model, items: this.collection.models }));
+        this.$('.tooltipped').tooltip({delay: 50});
```

Comment:
We shouldn't need this line, because as the Materialize docs explain: "Tooltips are initialized automatically"